src/shared/components/Graph/GraphVisualizer.less

```diff
@@ -0,0 +1,7 @@
+@import '../../lib.less';
+
+.graph-visualizer {
+  .interactiveArea();
+  height: auto;
+  min-height: 600px;
```

Comment:
why this number?

Reply:
The hight must be defined for this svg stuff, and I just think 600 looks pretty nice. So it's completely arbitrary

Reply:
Better indicate it explicitely in the code then :-)